torch/onnx/symbolic_opset11.py

```diff
@@ -85,17 +86,28 @@ def __interpolate(g, input, size, scale_factor, mode, align_corners):
     if 'cubic' in mode:
         mode = 'cubic'
     align_corners = sym_help._maybe_get_const(align_corners, 'b')
-    align_corners = False if sym_help._is_none(align_corners) else align_corners
+    align_corners = False if not isinstance(align_corners, bool) else align_corners
     coordinate_transformation_mode = "asymmetric" if mode == "nearest" \
         else "align_corners" if align_corners else "pytorch_half_pixel"
     # roi only takes effect whith coordinate_transformation_mode="tf_crop_and_resize"
     roi = g.op("Constant", value_t=torch.tensor([], dtype=torch.float32))
 
     if not sym_help._is_none(size) :
-        input_size = input.type().sizes()
-        input_size = g.op("Constant", value_t=torch.tensor(input_size[0:2], dtype=torch.int64))
-        is_scalar = ((sym_help._maybe_get_const(size, 't').dim() == 0))
+        input_size = g.op("Shape", input)
+        input_size = sym_help._slice_helper(g, input_size, axes=[0], ends=[2], starts=[0])
+        # in some cases size is not a packed list but size is a scalar
+        # We need to also verify that (sym_help._maybe_get_const(size, 't').dim() == 0)
+        # but this information is not always available. Try to get the dim,
+        # and if not assume that it is not a scalar.
+        try:
+            is_scalar = not sym_help._is_packed_list(size) and ((sym_help._maybe_get_const(size, 't').dim() == 0))
+        except AttributeError:
+            warnings.warn("Cannot verify if the output_size is a scalar while exporting interpolate. Assuming that it is not a scalar.")
+            is_scalar = not sym_help._is_packed_list(size)
+
         if is_scalar:
+            if not input.isCompleteTensor():
```

Comment:
nit: seems only rank info is needed, so this can be relaxed a bit.

Reply:
we access the dimension line 112. previously it was silently failing